### PR TITLE
FOGL-397: docstring and comments for each schedule process test 

### PR DESCRIPTION
**Pull request by oshadmon**
The following pull request is the addition of comments to each of the tests.

### CODE REVIEW

File: src/python/tests/core/test_scheduler.py [hunk 1]
```diff
@@ -79,21 +102,22 @@ class TestScheduler:
         interval_schedule.name = 'sleep10'
         interval_schedule.process_name = "sleep10"
 
-        await scheduler.save_schedule(interval_schedule)
+        await scheduler.save_schedule(interval_schedule)  # Save update on scheduler
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler.py]: I would avoid comments that are pretty much just method names. Sometimes these kinds of comments are useful on their own line to separate one section of the code from the other.

File: src/python/tests/core/test_scheduler.py [hunk 2]
```diff
@@ -308,27 +364,34 @@ class TestScheduler:
...
     async def test_get_schedule(self):
+        """Schedule gets retrieved
+        :assert:
+            Schedule is retrieved by id """
         scheduler = Scheduler()
 
         await scheduler.populate_test_data()
         await scheduler.start()
 
+        # Declare schedule
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler.py]: These kinds of comments that separate sections of a 'def' are useful for readability

File: src/python/tests/core/test_scheduler.py [hunk 3]
```diff
@@ -79,21 +102,22 @@ class TestScheduler:
...
         interval_schedule.process_name = "sleep10"
 
-        await scheduler.save_schedule(interval_schedule)
+        await scheduler.save_schedule(interval_schedule)  # Save update on scheduler
 
-        tasks = await scheduler.get_running_tasks()
         await asyncio.sleep(1)
-        assert len(tasks) == 0
+        tasks = await scheduler.get_running_tasks()
+        assert len(tasks) == 1  # assert a single task is running
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler.py]: I would do it like this instead
\# assert: a single task is running
tasks = await..
assert( ..)

**Comment by oshadmon** [comment]
@Terris-Dianomic - merge complete! Please review and either confirm or write more errors. 

Thanks!

File: src/python/tests/core/test_scheduler_REMOTE_5532.py [hunk 4]
```diff
@@ -0,0 +1,423 @@
+# -*- coding: utf-8 -*-
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler_REMOTE_5532.py]: Please delete this file

File: src/python/tests/core/test_scheduler_LOCAL_5532.py [hunk 5]
```diff
@@ -0,0 +1,479 @@
+# -*- coding: utf-8 -*-
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler_LOCAL_5532.py]: Please delete this file

File: src/python/tests/core/test_scheduler_BASE_5532.py [hunk 6]
```diff
@@ -0,0 +1,386 @@
+# -*- coding: utf-8 -*-
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler_BASE_5532.py]: Please delete this file

File: src/python/tests/core/test_scheduler_BACKUP_5532.py [hunk 7]
```diff
@@ -0,0 +1,524 @@
+# -*- coding: utf-8 -*-
```
Comment (Terris-Dianomic) [inline src/python/tests/core/test_scheduler_BACKUP_5532.py]: Please delete this file

**Comment by oshadmon** [comment]
My mistake, Fixed, and pushed.

**Pull request merged**